test/test_energy_storage_device.py

```diff
@@ -78,11 +88,15 @@ def test_consistency_linear_evolve_functions(self):
         df.evolve_one_time_step_linear_power(dt, p_fin,
                                              divisor=div,
                                              start_point=8.0)
-        # since power is not reported by dualfoil, must
-        # calculate with given values; leads to high error caused
-        # by significant roundoff when working with small currents;
-        # relax the error requirement
+
+        # BELOW: relaxing the error requirement
+        # REASON: Like above, power must be calculated with current
+        #   and voltage, which both have roundoff error. However, 
+        #   running a simulation with multiple retstarts likely
+        #   accumulates in roundoff errorsto account for a larger
+        #   error by comparison
```

Comment:
I am sure from experience that the test fails if I keep the error at the same value as with the constant evolve functions. Since the simulations run for the same amount of time (15 seconds), the only differences are the restarts and the fact that the value of the constant parameter changes with each restart.